security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+Integrity implies that the information does not get altered in transit.
+Availability ensures that users may expect the system to be functioning most of the time.
+
+The important properties of the Lightning Network are mostly centered around confidentiality and availability.
+Here are some of the properties we may want to ensure:
+
+* nobody except the sender and the receiver knows the payment amount
+* nobody can link senders and receivers
+* nobody can block an honest user from sending and receiving payments
+
+For each privacy goal and security model, there is a certain probability that an attacker succeeds.
+This probability depends on various factors, such as the size and the structure of the network.
+Other things equal, it is generally easier to attack a small network rather than a large one.
+Similarly, the more centralized the network is, the more capable an attacker can be if "central" nodes are under their control.
+Of course, the notion of "centralization" must be defined precisely to build security models around it, and there are many possible definitions of how centralized a network is.
+Finally, as a payment network, the LN depends on economic stimuli.
+The size and structure of fees affect the routing algorithm and, therefore, can either aid the attacker by forwarding most payments through their nodes or prevent this from happening.
+
+
+=== Anonymity set
+
+What does it mean to deanonymize someone?
+In simple terms, deanonymization implies linking some action with a person's "real world" identity, such as their name or physical address.
+In privacy research, the notion of deanonymization is more nuanced.
+First, we are not necessarily talking about names and addresses.
+Discovering someone's IP address or telephone number may also be considered deanonymization.
+A piece of information that allows linking a user's action to their previous actions is referred to as _identity_.
+Second, deanonymization is not binary.
+A user is not either fully anonymous or completely deanonymized.
+Instead, privacy research looks at anonymity compared to the _anonymity set_.
+
+The anonymity set is a central notion in privacy research.
+It refers to the set of identities such that, from an attacker's viewpoint, a given action could correspond to any of them.
+Consider a real-life example.
+Imagine you meet a person on a city street.
+What is their anonymity set from your point of view?
+If you don't know them personally, and without any additional information, their anonymity set roughly equals the city's population, including travelers.
+If you additionally consider their appearance, you may be able to roughly estimate their age and exclude the city residents who are obviously older or younger than the person in question from the anonymity set.
+Furthermore, if you notice that the person walks into the office of Company X using an electronic badge, the anonymity set shrinks to the number of Company X's employees and visitors.
```

Comment:
Awesome explanation! I think you can safely make up a company name (maybe StarBlocks is already mentioned in this book)?